Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WrapLayout/WrapLayoutPage.xaml.cs

```diff
@@ -40,6 +45,10 @@ public void OnXamlRendered(FrameworkElement control)
             {
                 repeater.ItemsSource = _items;
             }
+
+            _wrapScrollParent = control.FindChildByName("WrapScrollParent") as ScrollViewer;
+            var wrapLayoutParent = _wrapScrollParent?.FindDescendant<ItemsRepeater>();
+            _wrapLayout = wrapLayoutParent.Layout as WrapLayout;
```

Comment:
Can be replaced with 
```
_wrapLayout = repeater.Layout as WrapLayout;
```

Reply:
Thanks Shawn, I just fixed this. I also had to switch to the Visual Tree Helpers and the Logical Ones weren't running in Release Mode for some reason.